### ISSUE TITLE
Gui Builder: Res Image list is empty

### ISSUE THREAD

**Issue by kutoman** [codenameone/CodenameOne#2734]
when I try to pick an image (e.g. as icon) there is nothing to pick from in the "from res file" view even though my theme file does contain various images. 
Note: css is activated and I'm coding in intellij

**Comment by codenameone** [comment]
Do you have more than one res file?
Do you see errors in the command line: https://www.codenameone.com/blog/tip-track-designer-guibuilder-issues.html
If you open the res file in the designer do you see images in the image section?

**Comment by kutoman** [comment]
yes I have more than one res file as I need to have a seperate res file for localization since I activated css. yes I see various images over the designer.

When I run the builder over the command line there is no error printed out but I noticed that the builder is started with the wrong res file:
`resfile='file:/Users/User/Development/projects/MyProject/src/lang.res'`
it should be `theme.res` instead
